Tools/TestWebKitAPI/cocoa/TestWKWebView.mm

```diff
@@ -526,6 +527,9 @@ - (void)_setUpTestWindow:(NSRect)frame
 
 - (void)addToTestWindow
 {
+    if (!_hostWindow)
+        [self _setUpTestWindow:self.frame];
```

Comment:
We can return after this. _setUpTestWindow already runs the logic below.

Reply:
Ah! Good catch!